gradleSetup.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```

Comment:
I would it is not gradle setup it is more prepare jmeter properties.
When I read this file name I was thinking about installation of gradle.